src/app/common/subscriptions/SubscriptionDetails.vue

```diff
@@ -1,59 +1,66 @@
 <template>
   <div>
-    <div v-if="details.globalInstanceId || details.connectTime || details.disconnectTime">
+    <div v-if="details.globalInstanceId || details.controlPlaneInstanceId || details.connectTime || details.disconnectTime">
```

Comment:
Is this a bug fix?

Reply:
Perhaps. Looking at the current implementation, all uses of these fields are guarded with `v-if`s indicating that they can be empty/unset. If so, the top-level condition is indeed missing this check. However, I’m not sure if this field _can_ be empty/unset. So it’s cautionary, but I haven’t seen this actually be broken in practice.